src/configs/directory.rs

```diff
@@ -21,8 +19,7 @@ impl<'a> RootModuleConfig<'a> for DirectoryConfig<'a> {
             truncate_to_repo: true,
             fish_style_pwd_dir_length: 0,
             use_logical_path: true,
-            prefix: "in ",
-            style: Color::Cyan.bold(),
+            format: "[$path](cyan bold) ",
```

Comment:
Shall we add the `in ` prefix in the `format` string? I didn't see that prefix locally and my personal configuration didn't overwrite it. Hence to have similar results, I didn't add it back here.

Reply:
Ooh... You've landed on an interested edge-case we didn't account for. The first prefix on each line isn't shown.

Yeah, I think it's best if we remove it with format strings.

Reply:
Ah that would explain why I didn't see it locally!

What will happen to prefix/suffix support? Will this be removed once the formatter is fully implemented?

Reply:
Yep, that's the plan.